### PR TITLE
Update configure-block-at-first-sight-windows-defender-antivirus.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2218

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-antivirus/configure-block-at-first-sight-windows-defender-antivirus.md [hunk 1]
```diff
@@ -65,6 +65,9 @@ Block at first sight requires a number of settings to be configured correctly or
 
    ![Intune config](images/defender/intune-block-at-first-sight.png)
 
+> [!Warning]
+> Setting file blocking level to "High" will apply a strong level of detection. It may unlikely cause false-positive detection of legitimate files. In this case, use the option to [restore the quarantined files](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-antivirus/restore-quarantined-files-windows-defender-antivirus).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-antivirus/configure-block-at-first-sight-windows-defender-antivirus.md]:
```suggestion
> Setting the file blocking level to **High** will apply a strong level of detection. In the unlikely event that it causes a false positive detection of legitimate files, use the option to [restore the quarantined files](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-antivirus/restore-quarantined-files-windows-defender-antivirus).
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
your recommendations were successfully applied, thank you!

**Comment by mypil** [comment]
@Dansimp - Can you please assist to check, approve and merge the content of this PR based on issue #2218?

Thank you.

**Comment by mypil** [comment]
@Dansimp @AndreaBarr -  We would like to follow up. Can you please assist to check, approve and merge the content of this PR based on issue #2218?

Thank you.

**Pull request merged**